boards/DCM/Src/App/states/App_DriveState.c

```diff
@@ -1,20 +1,38 @@
+#include <stdlib.h>
+#include <math.h>
+#include "App_SharedMacros.h"
 #include "states/App_AllStates.h"
 #include "states/App_InitState.h"
 #include "App_SetPeriodicCanSignals.h"
+#include "configs/App_EfficiencyLUT.h"
 
-// TODO: Have separate maximum torque requests for motoring and generating
 #define MAX_TORQUE_REQUEST_NM (90.0f)
+#define RPM_TO_RADS(rpm) ((rpm) * (float)M_PI / 30.0f)
 
 void App_SetPeriodicCanSignals_TorqueRequests(struct DcmCanTxInterface *can_tx, struct DcmCanRxInterface *can_rx)
 {
-    float torque_request = 0.0f;
-
-    //              Accelerator Pedal Percentage
-    //  Torque =  -------------------------------  * MAX_TORQUE_REQUEST_NM
-    //                        100%
-    //
+    uint8_t lut_idx;
+    float   efficiency_estimate;
+    float   max_torque_request, torque_request;
+    float   bms_available_power = App_CanRx_BMS_AVAILABLE_POWER_GetSignal_AVAILABLE_POWER(can_rx);
+    float right_motor_speed_rpm = (float)abs(App_CanRx_INVR_MOTOR_POSITION_INFO_GetSignal_D2_MOTOR_SPEED_INVR(can_rx));
+    float left_motor_speed_rpm  = (float)abs(App_CanRx_INVL_MOTOR_POSITION_INFO_GetSignal_D2_MOTOR_SPEED_INVL(can_rx));
+
+    // Get the estimated efficiency for the current motor speed
+    lut_idx =
```

Comment:
for simplicity sake we should 
1) remove this variable
2) create a #define for the efficiency estimate and place it in this file
3) remove `EFFICIENCY_LUT_RPM_INTERVAL` in the header file (dont think its used anywhere after lut_index is removed)

we can keep the struct in the header file for now

Should be ready to merge after this

Reply:
left `EFFICIENCY_LUT_RPM_INTERVAL` incase we use it